test/example-types/tagsQuery.js

```diff
@@ -0,0 +1,164 @@
+let {
```

Comment:
There seem to be some missing test cases, like stripping out extra spaces (e.g. `a    b` should be `a b`). Did you go through all the unit tests for queryFilterNew that were in the app?

Reply:
Let me go back and review.